draft-iab-covid19-workshop.md

```diff
@@ -273,25 +273,27 @@ impact on networking. With large numbers of people working from home
 or otherwise depending on the network for their daily lives, network
 traffic volume has surged. Internet service providers and operators have
 reported a 20% traffic growth or more in a matter of weeks. Traffic at
-Internet Exchange Points (IXPs) is similarly on the rise. Most forms
+Internet Exchange Points (IXPs) is similarly on the rise, as were attacks targeting remote working infrastructure and platforms. Most forms
 of network traffic have seen an increase, with conversational
-multimedia traffic growing in some cases more than 200%. And user time
+multimedia traffic growing in some cases more than 200% and an explosion in COVID-19 related scams and phishing. And user time
 spent on conferencing services has risen by an order of magnitude on
 some conferencing platforms.
 
+Some themes of the workshop focused on an increase in Internet usage more generally, with reference to measurement and performance - where others focused on user experience and problems arise directly from the pandemic, such as home working and user behavioral changes.
+
 In general, the Internet has coped relatively well with this traffic
 growth, albeit not without some issues. For instance, some outages,
-video quality reduction, and other issues were reported. Nevertheless,
-it is interesting to see how the technology, operators and service
-providers have been able to respond to large and sudden changes in traffic
+video quality reduction, a new wave of attacks and scams, and other issues were reported. Nevertheless,
+it is interesting to see how the technology, operators, service
+providers and incident responders have been able to respond to large and sudden changes in traffic
```

Comment:
This was really just focusing on the Internet technology (which is applied by network operators) and not to be an exhausted list of all entities involved. Again I think a separate paragraph would be much better than trying to squeeze things in.

Reply:
```suggestion
providers have been able to respond to large and sudden changes in traffic
```